### DISCUSSION TITLE
关于小白遇到的一个问题

### DISCUSSION THREAD

**Discussion by yuyuyuka** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#17] [Q&A]
作者大大好，我4月底到5月初没玩了一段时间，今天回来发现更新后训练设施窗口不正常显示了（具体是偶尔隔了很久回合会出现一次一般不出现），尝试了很多方法都没有解决，群友使用的时候为正常，特意上来请教
![QQ截图20220510193452](https://user-images.githubusercontent.com/105284871/167619732-3a44d2ef-426c-4616-9836-10974c013c18.png)

**Comment by EtherealAO** [comment]
可能是某处处理有问题，请下载https://github.com/EtherealAO/UmamusumeResponseAnalyzer/releases/tag/v1.8.2 ，
在再次遇到此类BUG的时候把%localappdata%/UmamusumeResponseAnalyzer/packets目录下出现问题时的文件打包发送上来

**Comment by trojan-uma** [comment]
应该是窗口没有滚动刷新，如果你已经有操作但是软件窗口没有滚动的话请手动往下滚动
（但估计也是个bug，因为1.8以前的版本没有遇到过）

**Reply by EtherealAO** [reply]
这种情况是终端的BUG，大概至少从引入Spectre.Console的版本开始就有了（但是也不会是他们的问题）
